docs/docusaurus/docs/reference/learn/glossary.md

```diff
@@ -3,6 +3,25 @@ id: glossary
 title: "Glossary"
 ---
 
-import InProgress from '../../core/_core_components/_in_progress.md'
+[Action](/core/trigger_actions_based_on_results/trigger_actions_based_on_results.md): A component that integrates Great Expectations with other tools based on a Validation Result, like sending notifications based on the validation's outcomes.
```

Comment:
We could say that this is configurable on a Checkpoint, or maybe that is best left in the linked document.